c2corg_ui/static/partials/follow.html

```diff
@@ -0,0 +1,17 @@
+<div ng-if="followCtrl.canFollow()">
+
+  <div ng-show="followCtrl.followed" ng-click="followCtrl.toggle()"
+       class="float-button float-followed" tooltip-placement="left"
+       uib-tooltip="{{'No longer follow' | translate}}">
+    <span class="glyphicon glyphicon-star"></span>
+    <p class="float-button-text" translate>No longer follow</p>
+  </div>
+
+  <div ng-hide="followCtrl.followed" ng-click="followCtrl.toggle()"
+       class="float-button float-notfollowed" tooltip-placement="left"
+       uib-tooltip="{{'Follow' | translate}}">
```

Comment:
> Perhaps "Subscribe" in english?

In English it's "Follow/Unfollow". Maybe it's not clear to every user what "following a user" means.